libs/nest/core/src/lib/validators/isLocale.decorator.ts

```
@@ -0,0 +1,29 @@
+import { registerDecorator, ValidationOptions } from 'class-validator'
+import { supportedLocales } from '@island.is/shared/constants'
+import { Locale } from '@island.is/shared/types'
+
+export const IsLocale = (validationOptions?: ValidationOptions) => {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  return (object: Object, propertyName: string) => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)